scopes/workspace/workspace/install.cmd.tsx

```diff
@@ -46,11 +48,12 @@ export default class InstallCmd implements Command {
     this.logger.console(`Resolving component dependencies for workspace: '${chalk.cyan(this.workspace.name)}'`);
     const installOpts: WorkspaceInstallOptions = {
       variants: options.variants,
-      lifecycleType: options.type,
+      lifecycleType: options.addMissingPeers ? 'peer' : options.type,
```

Comment:
I don't think you really need to bind it here
later in the workspace.install you can make sure everything comes from the missing peers are added as peer.
Doing it here means that if I'm doing
`bit install lodash --add-missing-peers` will add lodash to my peer instead of to my regular deps

Reply:
Throwing an exception in such case for now.